content/concepts/module-resolution.md

```diff
@@ -24,26 +24,26 @@ The dependency module can be from the application code or a third party library.
 * Absolute paths
 
 ```js
-require("/home/me/file");
-require("C:\\Home\\me\\file");
+import "/home/me/file";
+import "C:\\Users\\me\\file";
```

Comment:
Do you mean something like `require(__dirname + "/path")`? Or what do you mean with `placeholders`?